plugins/outputs/opentelemetry/opentelemetry_test.go

```diff
@@ -66,13 +66,7 @@ func TestOpenTelemetry(t *testing.T) {
 		time.Unix(0, 1622848686000000000))
 
 	err = plugin.Write([]telegraf.Metric{input})
-	if err != nil {
-		// TODO not sure why the service returns this error, but the data arrives as required by the test
-		// rpc error: code = Internal desc = grpc: error while marshaling: proto: Marshal called with nil
-		if !strings.Contains(err.Error(), "proto: Marshal called with nil") {
-			assert.NoError(t, err)
-		}
-	}
+	assert.NoError(t, err)
```

Comment:
Can you please use `require.NoError` as we do prefer this over `assert` if possible.